src/units/abilities.cpp

```diff
@@ -563,27 +563,41 @@ template std::pair<int, map_location> unit_ability_list::get_extremum<std::great
 
 namespace {
 
+    struct special_match
+    {
+        std::string tag_name;
+        const config* cfg;
+    };
+
```

Comment:
Fix the indentation:

```suggestion
	struct special_match
	{
	    std::string tag_name;
	    const config* cfg;
	};

```